filebeat/Jenkinsfile.yml

```
@@ -49,6 +49,19 @@ stages:
                 - "macosTest"
             tags: true         ## for all the tags
         stage: extended
+    macosM1:
+        mage: "mage build unitTest"
+        platforms:             ## override default label in this specific stage.
+          - "orka && darwin && aarch64"
+        when:                  ## Override the top-level when.
+            comments:
+                - "/test auditbeat for macos-m1"
```

I believe this should be Filebeat:
```suggestion
                - "/test filebeat for macos-m1"
```